tools/osbuilder/image-builder/image_builder.sh

```diff
@@ -402,6 +419,27 @@ create_rootfs_image() {
 
 	info "Copying content from rootfs to root partition"
 	cp -a "${rootfs}"/* "${mount_dir}"
+
+	if [ "${SELINUX}" == "yes" ]; then
+		if [ "${AGENT_INIT}" == "yes" ]; then
```

Comment:
I'm not sure if `AGENT_INIT = yes ` equals using an initrd image.

Reply:
Sorry, I made a mistake. It's actually is that guest SELinux does not work with the agent init (`AGENT_INIT=yes`), so I fixed the comment and documentation.